package.json

```diff
@@ -2,22 +2,25 @@
   "name": "cozy-notes",
   "version": "1.17.0",
   "scripts": {
-    "tx": "tx pull --all || true",
-    "lint": "yarn lint:js && yarn lint:styles",
-    "lint:js": "cs lint '{src,test}/**/*.{js,jsx}'",
+    "build:browser:dev": "cs build --browser --development --ts",
+    "build:browser": "cs build --browser --ts",
+    "build:mobile": "cs build --mobile --ts",
+    "build": "yarn build:browser",
+    "cozyPublish": "cs publish --token $REGISTRY_TOKEN --build-commit $(git rev-parse ${DEPLOY_BRANCH:-build})",
+    "deploy": "git-directory-deploy --directory build/ --branch ${DEPLOY_BRANCH:-build} --repo ${DEPLOY_REPOSITORY:-https://cozy-bot:$GITHUB_TOKEN@github.com/cozy/cozy-notes}",
+    "install:clean:start": "yarn install:clean && yarn start",
+    "install:clean": "rm -rf node_modules && rm -f yarn.lock && yarn install",
+    "lint:js": "cs lint '{src,test}/**/*.{js,jsx,ts,tsx}'",
     "lint:styles": "stylint src/styles --config ./.stylintrc",
+    "lint": "yarn lint:js && yarn lint:styles",
     "prebuild": "yarn lint",
-    "build": "yarn build:browser",
-    "build:browser:dev": "cs build --browser --development",
-    "build:browser": "cs build --browser",
-    "build:mobile": "cs build --mobile",
-    "watch": "yarn watch:browser",
+    "stack:docker:dev": "COZY_STACK_HOST=cozy.localhost docker run --rm -it -p 8080:8080 -p 5984:5984 -e COZY_DISABLE_CSP=1 -v $(pwd)/build:/data/cozy-app/app cozy/cozy-app-dev",
+    "start": "cs start --browser --ts",
+    "test": "env NODE_ENV='test' cs test --verbose",
+    "tx": "tx pull --all || true",
     "watch:browser": "cs watch --browser",
     "watch:mobile": "cs watch --mobile",
-    "start": "cs start --browser",
-    "deploy": "git-directory-deploy --directory build/ --branch ${DEPLOY_BRANCH:-build} --repo ${DEPLOY_REPOSITORY:-https://cozy-bot:$GITHUB_TOKEN@github.com/cozy/cozy-notes}",
-    "test": "env NODE_ENV='test' cs test --verbose",
-    "cozyPublish": "cs publish --token $REGISTRY_TOKEN --build-commit $(git rev-parse ${DEPLOY_BRANCH:-build})"
+    "watch": "yarn watch:browser"
```

Comment:
Since there is a lot of new commands can you update the readme to explain how to use the app please ?

Reply:
Yes of course

Reply:
Updated